rules/alerting.go

```diff
@@ -105,12 +105,8 @@ type AlertingRule struct {
 	holdDuration time.Duration
 	// Extra labels to attach to the resulting alert sample vectors.
 	labels model.LabelSet
-	// Short alert summary, suitable for email subjects.
-	summary string
-	// More detailed alert description.
-	description string
-	// A reference to a runbook for the alert.
-	runbook string
+	// Non-identifying key/value pairs.
+	annotations model.LabelSet
```

Comment:
They have the same validation constraints. I had them as a separate thing initially in Alertmanager. But it was just cluttering the code.

As far as implementation is concerned, they are the same kind of key/value pairs.